src/serac/physics/tests/solid.cpp

```diff
@@ -136,8 +136,11 @@ void functional_solid_spatial_essential_bc()
   solid_solver.setMaterial(mat);
 
   // Set up
-  auto zero_vector   = [](const mfem::Vector&, mfem::Vector& u) { u = 0.0; };
-  auto zero_scalar   = [](const mfem::Vector&) { return 0.0; };
+  auto zero_vector = [](const mfem::Vector&, mfem::Vector& u) { u = 0.0; };
+
+  // We want to test both the scalar displacement functions including the time argument and the scalar displacement
+  // functions without
+  auto zero_scalar   = [](const mfem::Vector&, double) { return 0.0; };
```

Comment:
it might be worth only supporting the function signatures that include time, so there's half as many b.c. overloads to deal with

Reply:
Agreed. I'll open this in an issue and complete it as a follow-on PR.